File: JavaScript/JavaScriptProgrammingBasics/L06_NestedLoops/Lab/P06_Building.js
function building(input) {
    let index = 0;
    const floors = parseInt(input[index++]);
    const roomsPerFloor = parseInt(input[index++]);

    for (let f = floors; f >= 1; f--) {
        let symbol = ``, line = ``;
        if (f == floors) {
            symbol = `L`
        }
        else if (f % 2 != 0) {
            symbol = `A`
        }
        else if (f % 2 == 0) {
            symbol = `O`
        }
        for (let r = 0; r < roomsPerFloor; r++) {
            line += ` ${symbol}${f}${r}`;
        }
        console.log(line.trim());
    }
}

File: Python/M01_ProgrammingBasics/L06_NestedLoops/Lab/Solutions/P06_Building.py
floors = int(input())
rooms = int(input())

for x in range(floors, 0, -1):
    room_type = ""
    if x == floors:
        room_type = "L"
    elif x % 2 == 1:
        room_type = "A"
    elif x % 2 == 0:
        room_type = "O"

    for y in range(rooms):
        print(f"{room_type}{x}{y}", end=" ")
    print()
